src/duckula/handler.clj

```diff
@@ -127,7 +127,7 @@ It depends on a component implementing  duckula.prococol/Monitoring protocol
                   (monitoring/on-success monitoring success-key {:body body :status status})
                   (monitoring/on-error monitoring error-key))
                 response)
-              (catch Exception err
+              (catch Throwable err
```

Comment:
@lukaszkorecki we need this to catch assert failures etc (I assume we want to handle those as failure as well?)

Reply:
Yep!